cedar-policy-core/src/ast/name.rs

```diff
@@ -167,6 +178,34 @@ impl FromNormalizedStr for Name {
     }
 }
 
+struct NameVisitor;
+
+impl<'de> serde::de::Visitor<'de> for NameVisitor {
+    type Value = Name;
+
+    fn expecting(&self, formatter: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        formatter.write_str("a name consisting of a path and id, separated by `::`")
```

Comment:
maybe this comment could make it clear that the path (and thus the `::`) is optional?  Not sure if that's easy to concisely convey though

Reply:
Updated to `"a name consisting of an optional namespace and id"`. Couldn't find a concise way to work in the `::` but open to suggestions.